components/screens/DailyRidesScreen/DailyRidesScreen.js

```diff
@@ -85,7 +92,15 @@ export default class App extends React.Component {
       <SafeAreaView style={Styles.container}>
         <FlatList
           data={this.state.data}
-          renderItem={item => this.renderItemComponent(item)}
+          renderItem={(item) => {
+            const d=item.item.date;
```

Comment:
Hi @CalebMcOlin , I thought closely about this before deciding to just use "d" for the datestring we are pulling from each item. I feel that since the use of variable is limited to just  a few lines, it is fine as is because it makes the block easier to read. I will add a comment pointing to what "d" stands for.
```
            const d=item.item.date;
            let rideDate=''; 
            d[10]===',' ? rideDate=d.slice(0,3)+d.slice(4,10)+d.slice(11,16) : 
              rideDate=d.slice(0,3)+d.slice(4,11)+d.slice(12,17);
            if (rideDate===today){
              return this.renderItemComponent(item)};
```